### DISCUSSION TITLE
일부 유저만 채팅을 못하도록 제한할 수 있나요?

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#714] [5. Q&A]
멀티 게임에서 유저가 탈락하면 해당유저만 채팅기능을 비활성화 시킬 수 있을까요?
탈락하지 않은 유저들은 계속 채팅을 칠 수 있어야만 합니다.
그리고 탈락한 유저들은 해당 방에서 나가지는 것이 아닌 관전상태입니다.
그래도 탈락 유저만 채팅을 블락할 수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 채팅은 월드 공통 기능이므로 현재로서는 커스텀할 수 있는 API를 제공하지 않습니다.
감사합니다.
